src/Frontend/Core.hs

```diff
@@ -336,6 +337,26 @@ percentVotes idea numVoters vv = {- assert c -} v
           then 100
           else (numVotes idea vv * 100) `div` numVoters
 
+callToActionOnList
+    :: (Foldable t, Monad m)
+    => HtmlT m ()
+    -> (HtmlT m () -> HtmlT m ())
+    -> (a -> HtmlT m ())
+    -> t a
+    -> HtmlT m ()
+callToActionOnList onCta wrapItems onItem values
+  | null values = div_ [class_ "container-not-found"] $ onCta
+  | otherwise   = wrapItems $ mapM_ onItem values
```

Comment:
I've grown to like infixing `mapM_`, but I don't insist yet on adding it to the style guide.  (-: